src/main/java/com/songrLab/songr/SongrApplication.java

```diff
@@ -1,8 +1,10 @@
 package com.songrLab.songr;
 
-import com.songrLab.songr.Models.Album;
-import com.songrLab.songr.Repositories.AlbumRepo;
-import org.springframework.beans.factory.annotation.Autowired;
+import com.songrLab.songr.models.Album;
```

Comment:
I'm noticing some capitalization issues here. When I look at the repo on GitHub, it shows that you have both a Models directory and a models directory, as well as a Repositories directory and a repositories directory. 

I'm not able to run the app, because it's having trouble understanding which directory it should be pulling from. Best practice is to capitalize directory names; either way, you'll need to get rid of the extra directory and double-check your imports to make sure they're pointing in the right direction.

Reply:
Thanks Lucy! I thought I worked through that and then I had some git conflicts when I pushed up between two branches. I merged/resolved them with Tony's help but it's possible the pull request I submitted isn't correct yet.